### ISSUE TITLE
Item thumbnails have different projection than STAC Explorer expects

### ISSUE THREAD

**Issue by j08lue** [NASA-IMPACT/veda-backend#420]
The new item thumbnails are amazing for getting a visual impression of the data in a client application like STAC Explorer!

Unfortunately, for many collections in our VEDA Earthdata catalog which are on global lat/lon grids (e.g. `fldas-soil-moisture-anomalies`), they do not mount well on the preview map in STAC Explorer:

![image](https://github.com/user-attachments/assets/8bc0f585-d64d-493a-a4cb-c341d9dcd61f)

https://staging.openveda.cloud/stac/collections/fldas-soil-moisture-anomalies/items/FLDAS_NOAH01_SoilMoi00_10cm_tavg_C_GL_MA_ANOM202307_20230701?.language=en&.asset=asset-rendered_preview

I guess the thumbnail is not reprojected and just a dump of the native grid (equirectangular, i.e. regular lat/lon), while the little preview map is using Web Mercator. Preview thumbnails in native map projection are more informative than reprojected ones, I'd say, but it just does not look good...

Btw, Planetary Computer STAC / STAC Explorer has the same issue with preview thumbnails for global gridded datasets, e.g. https://radiantearth.github.io/stac-browser/#/external/planetarycomputer.microsoft.com/api/stac/v1/collections/noaa-cdr-ocean-heat-content/items/ocean-heat-content-2020-Q2-2000m?.asset=asset-rendered_preview

Can we fix this somehow, so the preview images show nicely on the STAC Explorer map - or we disable this map, because it will be wrong for most datasets? (btw, no is also an acceptable answer...)

## Acceptance criteria

- [ ] STAC Explorer does not show a wrong map representation of the preview thumbnails

**Comment by anayeaye** [comment]
I wonder if setting `dst_crs` in the renders params would address this?

**Comment by j08lue** [comment]
Nice, yes, the preview does get reprojected with `dst_crs=epsg:3857`. Not sure the bounding box is well preserved, though? 

https://staging.openveda.cloud/api/raster/collections/fldas-soil-moisture-anomalies/items/FLDAS_NOAH01_SoilMoi00_10cm_tavg_C_GL_MA_ANOM202307_20230701/preview.png?bidx=1&assets=cog_default&rescale=-0.3%2C0.3&colormap_name=rdbu&dst_crs=epsg:3857

This one in particular gets weirdly expanded towards the north.

![image](https://github.com/user-attachments/assets/6a89ab3d-0474-42a8-8b55-d2d7b75fda1e)

**Comment by anayeaye** [comment]
@j08lue that is looooooong. 4326? https://staging.openveda.cloud/api/raster/collections/fldas-soil-moisture-anomalies/items/FLDAS_NOAH01_SoilMoi00_10cm_tavg_C_GL_MA_ANOM202307_20230701/preview.png?bidx=1&assets=cog_default&rescale=-0.3%2C0.3&colormap_name=rdbu&dst_crs=epsg:4326

**Comment by anayeaye** [comment]
🤔 I tried putting the dst_crs epsg:3857 in the collection renders config just to see if the browser handled it any better. Nope
<img width="665" alt="Screenshot 2024-08-12 at 3 09 43 PM" src="https://github.com/user-attachments/assets/ac696899-d5fb-4fb5-8001-05427b316feb">

**Comment by j08lue** [comment]
Yes, all the map does is take the PNG and place it within the given bbox (from STAC). So the long PNG gets used as-is.

**Comment by vincentsarago** [comment]
👀 

I'm pretty sure the preview should be in EPSG:4326, the issue then is to know which bbox (in epsg4326) correspond to image returned by the service

**Comment by j08lue** [comment]
Which bbox does STAC Browser use to mount the image? And why do image and that bbox not match?

**Comment by smohiudd** [comment]
Can we manually deploy a pre-release version of stac browser that is using a different mapping framework other than leaflet?

**Comment by j08lue** [comment]
We are currently working through a similar problem for the UK EO Data Hub Resource Catalogue - we want to mount thumbnails (of products that cannot be rendered on the fly) on a map, so users looking for suitable items can get a good preview of them.

Some collections had item thumbnails that are in native image projection. We chose a hack to derive a rotated and skewed (trapezoid) bounding box from the item geometry. But it is not clean.

It would be great if STAC had a spec for how to provide thumbnails and anchor points. Obviously, if the only anchor points are the lat/lon bbox, the image needs to be non-rotated in geographic projection.

If thumbnails can have arbitrary projections, e.g. the native image projection, we need more information in STAC.

For example, MapLibre can mount rotated images, but [requires four image corner coordinates](https://maplibre.org/maplibre-gl-js/docs/API/type-aliases/Coordinates/) to mount a non-georeferenced `ImageSource` to the map.

**Comment by j08lue** [comment]
Why do we expect better display of thumbnails from the OpenLayers map in the upcoming STAC Browser version, @smohiudd?

**Comment by smohiudd** [comment]
@j08lue we don't know for sure but will try anything at this point! It may not work.

> If thumbnails can have arbitrary projections, e.g. the native image projection, we need more information in STAC.

What other information do we need or can be derived from stac meta data to help with this?

**Comment by j08lue** [comment]
But thumbnails are also used in all kinds of contexts, not just maps, so maybe we should stick to PNGs.

Then, we should make sure the thumbnails fit he bbox that is stored in STAC. TiTiler's `/preview` may be inferring the bbox differently from what we have stored in STAC.

 We could use TiTiler's [`/bbox`](https://openveda.cloud/api/raster/docs#/STAC%20Item/bbox_image_collections__collection_id__items__item_id__bbox__minx___miny___maxx___maxy___format__get) endpoint instead to be sure thumbnail and bbox always match.
